### ISSUE TITLE
[ArchiveContent] Move files via drag'n'drop

### ISSUE THREAD

**Issue by FreakyBytes** [SemsTestOrg/combinearchive-web#6]
**component:** code
**owner:** martinP
**reporter:** martinP
**created:** 2014-07-31 08:08:16
**milestone:** 
**type:** task
**version:** 
**keywords:** 

add a [New Folder] button and implement a smart drag'n'drop mechanism for the file tree and limit the drop to folders only

**Comment by FreakyBytes** [comment]
**time:** 2014-07-31 08:08:44
**author:** martinP

**Comment by FreakyBytes** [comment]
**time:** 2014-07-31 08:08:44
**author:** martinP

Updated **description** to **add a [New Folder] button and implement a smart drag'n'drop mechanism for the file tree**

**Comment by FreakyBytes** [comment]
**time:** 2014-07-31 08:31:16
**author:** martinP

**Comment by FreakyBytes** [comment]
**time:** 2014-07-31 08:31:16
**author:** martinP

Updated **description** to **add a [New Folder] button and implement a smart drag'n'drop mechanism for the file tree and limit the drop to folders only**

**Comment by FreakyBytes** [comment]
**time:** 2014-08-16 23:18:40
**author:** mp487 <martin.peters3@uni-rostock.de>


In changeset:"68caae85a4f9566f6c87c1ec5ecab38145adde6c"]:
```CommitTicketReference repository="" revision="68caae85a4f9566f6c87c1ec5ecab38145adde6c"

added new folder button to ArchiveView [fixes #44] and preparation for
[see #5]

```

**Comment by FreakyBytes** [comment]
**time:** 2014-08-16 23:22:55
**author:** martinP
